src/middlewared/middlewared/plugins/etc.py

```diff
@@ -151,7 +151,8 @@ def __init__(self, *args, **kwargs):
                         changes = True
 
             if not os.path.exists(outfile):
-                continue
+                with open(outfile, 'w') as f:
+                    f.write(rendered)
```

Comment:
I'd agree with you but left it as is, since the general case is probably the permissions will be set and set it anyway. However, it's fine if added here as well.

Reply:
and, updated!